### PR TITLE
Change deps to go mod and update libs

### PR DESCRIPTION
**Pull request by martinsrso**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -34,4 +39,8 @@ ENV USE_BASICAUTH false
 ENV ARKADIKO_BASICAUTH_USERNAME ""
 ENV ARKADIKO_BASICAUTH_PASSWORD ""
 
-CMD /go/bin/arkadiko start --bind 0.0.0.0 --port 8890 --config /home/arkadiko/local.yml
+ENTRYPOINT ["./arkadiko", "start", "--bind", "0.0.0.0", "--port", "8890", "--config", "/config/local.yml"]
+EXPOSE 8890
+
+LABEL app=arkadiko
+LABEL maintainer='TFG CO <backend@tfgco.com>'
```
Comment (lmsilva-wls) [inline Dockerfile:46]: These were already defined on lines 9 and 10

File: Dockerfile [hunk 2]
```diff
@@ -34,4 +39,8 @@ ENV USE_BASICAUTH false
 ENV ARKADIKO_BASICAUTH_USERNAME ""
 ENV ARKADIKO_BASICAUTH_PASSWORD ""
 
-CMD /go/bin/arkadiko start --bind 0.0.0.0 --port 8890 --config /home/arkadiko/local.yml
+ENTRYPOINT ["./arkadiko", "start", "--bind", "0.0.0.0", "--port", "8890", "--config", "/config/local.yml"]
+EXPOSE 8890
+
+LABEL app=arkadiko
```
Comment (leeohaddad) [inline Dockerfile:45]: Is it expected that this line and line below are duplicate?

**Review by leeohaddad — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (leeohaddad) [reply Dockerfile:46]: Looks like we made the same comment at the same time 😅

File: Dockerfile (hunk 2 above)
Comment (martinsrso) [reply Dockerfile:45]: Yes because they are two images. One for build and one for application.

File: Dockerfile (hunk 1 above)
Comment (martinsrso) [reply Dockerfile:46]: Yes because they are two images. One for build and one for application.

File: Dockerfile (hunk 2 above)
Comment (leeohaddad) [reply Dockerfile:45]: 👌

**Review by lmsilva-wls — APPROVED** [review]

**Pull request merged**